setup.py

```diff
@@ -5,17 +5,17 @@
 import fnmatch
 import sys
 import shlex
-from Cython.Distutils import build_ext
```

Comment:
What does this change do?

Reply:
Related to:

https://github.com/healpy/healpy/pull/686

I was getting that error again, so I had to change that. I added reference to the commit now.